example/src/locale.rs

```diff
@@ -10,6 +10,7 @@ pub mod ffi {
     impl ICU4XLocale {
         /// Construct an [`ICU4XLocale`] from a locale identifier represented as a string.
         #[diplomat::attr(supports = constructors, constructor)]
+        #[diplomat::demo(input(name(label = "Locale Name")))]
```

Comment:
thought: would be cool to support a default value

thought 2: would be cool to support the ability to pick from a small list _plus_ an other option the way the current demo does. very very unimportant